conjure-java-core/src/test/resources/errors-for-endpoints.yml

```diff
@@ -0,0 +1,22 @@
+types:
```

Comment:
I didn't want to re-use the errors in `example-errors.yml`. If I used any of those errors in an endpoint, the `ErrorGenerator` would not create the static `ServiceException` factory methods for them. This would conflict with the generated error classes. I don't want to change the current `ErrorGenerator` tests.